plugins/gatsby-ample-pages/gatsby-node.js

```diff
@@ -3,43 +3,19 @@ const path = require("path")
 exports.createPages = ({ graphql, actions }) => {
```

Comment:
This file is a great example of the benefit gatsby-remark-ample provides us. Look at how much simpler the logic is to building out pages!